conf/frps_full.ini

```diff
@@ -71,3 +71,10 @@ tcp_mux = true
 
 # custom 404 page for HTTP requests
 # custom_404_page = /path/to/404.html
+
+
+# set promes_addr and promes_port to get prometheus monitor of frps
+# promes_addr's default value is same with bind_addr
+# prometheus exporter is available only if promes_port is set
+promes_addr = 0.0.0.0
```

Comment:
Reuse dashboard addr?

Reply:
Keep them separated. Because the Prometheus server won't depend on the dashboard server. The `dashboard_addr` may be turned off.

Reply:
We can use same port, but register different http handlers according to the demand.